externals/footer.phtml

```diff
@@ -9,7 +9,15 @@
             alt="Valid XHTML + RDFa" height="31" width="88" /></a>
         </p>
       </div>
-      <p><b>Copyright &copy; 2001-2018 The GnuCash Project</b></p>
+      <p><b>Copyright &copy; 2001–
+        <script type="text/javascript">
```

Comment:
Why not in PHP, the source language of this file?

Reply:
I'm no programmer, sorry. This is a snippet I used on some webpages i did and it worked there - so I figured it could work here as well. 

If PHP is better, don't hesitate to change my file.

Reply:
I've just done a quick research. Would this work?

```
      <p><b>Copyright &copy; 2001–<?= date("Y"); ?> The GnuCash Project</b></p>
```
If yes, I could do the change. Can I change the commited file, or do I have to do a new commit?

Reply:
Right.

After some reordering and rewordening everything is committed (commits 89d079b-44316e0)
Thanks!

Reply:
> I've just done a quick research. Would this work?
> 
> ```
>       <p><b>Copyright &copy; 2001–<?= date("Y"); ?> The GnuCash Project</b></p>
> ```
> 
> If yes, I could do the change. Can I change the commited file, or do I have to do a new commit?

Hm, I don't see my last answer,  so again:
Right!

After some reordering, rewordening and applying your suggestion, everything is commkitted.

Thanks!